### PR TITLE
performance regression analyzer initial version

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

File: utils/results_analyze.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+    @staticmethod
+    def render_to_html(res):
+        loader = jinja2.FileSystemLoader(os.path.dirname(os.path.abspath(__file__)))
+        env = jinja2.Environment(loader=loader, autoescape=True)
+        template = env.get_template('results.html')
+        html = template.render(dict(res_list=res))
+        return html
+
+    def send_email(self, subject, content, html=True):
```
Comment (amoskong) [inline utils/results_analyze.py]: Can we put email related functions to a single util file?
`utils/email.py`

**Comment by lmr** [comment]
Typo, 'pregression' -> 'regression', 'analyser' -> 'analyzer'

File: utils/results_analyze.py (hunk 1 above)
Comment (larisau) [inline utils/results_analyze.py]: sure, I'll separate it, so we could use it in other places

**Comment by roydahan** [comment]
Still need to work on the logic of what we compare and what we call a regression.

**Comment by amoskong** [comment]
https://en.wikipedia.org/wiki/Student%27s_t-test
In the past, we used T-test to insure a regression in netperf testing.

**Comment by larisau** [comment]
thanks, I'll take a look on it

**Pull request merged**